src/tests/ftest/daos_test/daos_core_test_dfs.yaml

```diff
@@ -50,11 +50,12 @@ dmg:
     allow_insecure: True
 daos_tests:
   Tests: !mux
-    test_u:
-      daos_test: u
-      num_clients: 1
-      test_name: DFS unit tests
-      test_timeout: 100
+# XXX Disabled until DAOS-6090 is fixed.
```

Comment:
I wonder if there is a way to apply https://github.com/daos-stack/daos/blob/e8b0b93f0b7f49bc07ab30b681a06bb3e623337c/src/tests/ftest/util/apricot/apricot/test.py#L62 to a single variant.  @phender any thoughts on this?

Reply:
I actually just landed a fix from Mohamad for this.